Source/WebKit/UIProcess/Automation/WebAutomationSession.cpp

```diff
@@ -477,11 +477,11 @@ void WebAutomationSession::switchToBrowsingContext(const Inspector::Protocol::Au
     m_client->requestSwitchToPage(*this, *page, [page = Ref { *page }, callback = WTFMove(callback)]() {
         page->setFocus(true);
 
-        callback->sendSuccess();
+        callback({ });
```

Comment:
i wonder if we should have a `Success` placeholder value or something like that so that it's a little clearer than `{ }`

ideally we'd have commands with `void` results not require any parameters, but not sure if there's a simple way of doing that with `Function` 🤔

Reply:
I was not able to think up with a cleaner solution in C++ other than `CommandResult<void>`.

`Success` would need to be an associated type or something, since the type of the Expected<T> differs.

Per EWS, in some cases it seems like the initializer list is too hard for GCC to infer the types, so in those cases we need an explicit `expected(T)`.